includes/managers/elements.php

```diff
@@ -298,6 +298,14 @@ private function init_categories() {
 			],
 		];
 
+		if ( Plugin::instance()->experiments->is_feature_active( 'container' ) ) {
+			$this->categories = array_merge_recursive( [
+				'layout' => [
+					'title' => esc_html__( 'Layout', 'elementor' ),
+				],
```

Comment:
I think for future use of this Layout outside of the context of the Flex Container, it might be an idea to remove the 'container experiment active' check, and to add this parameter: 

'hideIfEmpty' => true,

In this way the Layout will be hidden when the container is not active, but the usuage is extended beyond the container experiment.

Reply:
Works. Done.